confluent-registry-schema/1.0/modules/ROOT/pages/confluent-schema-registry-connector-examples.adoc

```diff
@@ -0,0 +1,345 @@
+= Confluent Schema Registry 1.0 Examples - Mule 4
+
+This example shows how to publish and consume avro messages by integrating
+Confluent Schema Registry with Kafka Connector to reduce the size of the
+messages. The example contains two flows:
+
+. The first flow generates an avro message containing the schema and sends it
```

Comment:
I would also mention that the Transform Message component generates an avro message containing the schema embeded in it and that the Replace Schema With Id operation replaces the schema embedded in the message with and id from the Confluent Schema Registry

Reply:
done